pkg/gofr/gofr.go

```diff
@@ -164,13 +164,19 @@ func (a *App) Run() {
 }
 
 // readConfig reads the configuration from the default location.
-func (a *App) readConfig() {
+func (a *App) readConfig(isAppCMD bool) {
```

Comment:
Needed to have a new flag `isAppCMD` because we want a **FileLogger** in case of a CMD app and a **normal logger i**n case of a normal app.